cmd/config.go

```diff
@@ -31,8 +31,9 @@ type Config struct {
 
 // Project holds information about a GitLab project
 type Project struct {
-	Name string
-	Refs string
+	Name            string
+	Refs            string
+	ExcludeArchived bool `yaml:"exclude_archived"`
```

Comment:
I am not convinced there is a use case for it on single project configuration 🤔

Reply:
Sorry for the slow response.  Yeah, I wasn't 100% sure about that either, I figured I'd add it for completeness, but it would be weird to explicitly ask for project A but also say "but only if it's not archived"...  I should get a chance this evening to remove that bit.

Reply:
I've updated and reworded the option as I got it the wrong way round first time.